Makefile

```diff
@@ -1,113 +1,240 @@
-###########################
-# Configuration Variables #
-###########################
-ORG := github.com/operator-framework
-PKG := $(ORG)/combo
-VERSION_PATH := $(PKG)/pkg/version
-GIT_COMMIT := $(shell git rev-parse HEAD)
-CONTROLLER_GEN := $(Q)go run sigs.k8s.io/controller-tools/cmd/controller-gen
-GO_BUILD := $(Q)go build
-PKGS := $(shell go list ./...)
-COMBO_VERSION :=  $(shell git describe --match 'v[0-9]*' --tags --always)
-
-
-# Binary build options
-KUBERNETES_VERSION=v0.22.2
-
-# Container build options
-IMAGE_REPO=quay.io/operator-framework/combo
-IMAGE_TAG=latest
-IMAGE=$(IMAGE_REPO):$(IMAGE_TAG)
-
-# kernel-style V=1 build verbosity
-ifeq ("$(origin V)", "command line")
-  BUILD_VERBOSE = $(V)
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= 0.0.1
+
+# CHANNELS define the bundle channels used in the bundle.
+# Add a new line here if you would like to change its default config. (E.g CHANNELS = "candidate,fast,stable")
+# To re-generate a bundle for other specific channels without changing the standard setup, you can:
+# - use the CHANNELS as arg of the bundle target (e.g make bundle CHANNELS=candidate,fast,stable)
+# - use environment variables to overwrite this value (e.g export CHANNELS="candidate,fast,stable")
+ifneq ($(origin CHANNELS), undefined)
```

Comment:
This Makefile has a bunch of stuff in it that we haven't made the decision to use yet; e.g. `kustomize`.